src/cgame/cg_demobackwardscompat.c

```diff
@@ -0,0 +1,33 @@
+#include "cg_local.h"
+
+static qboolean CG_DemoVersionBelow(int major, int minor, int patch)
+{
+	if (cg.demoVersion.major == 0 && cg.demoVersion.minor == 0 && cg.demoVersion.patch == 0)
+	{
+		return qfalse;
+	}
+
+	if (cg.demoVersion.major < major)
+	{
+		return qtrue;
+	}
+	else if (cg.demoVersion.minor < minor)
+	{
+		return qtrue;
+	}
+	else if (cg.demoVersion.patch < patch)
+	{
+		return qtrue;
+	}
+
+	return qfalse;
+}
+
+void CG_DemoBackwardsCompatInit()
+{
+	if (CG_DemoVersionBelow(2, 83, 0))
```

Comment:
isn't it more correct to do version<=2.82.1 ? then it would not need to assume what next git tag is

Reply:
doesn't matter in this case because we already know the next release number